src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -132,5 +132,20 @@
             <argument type="collection" /> <!-- All type guessers are stored here by FormPass -->
             <tag name="console.command" command="debug:form" />
         </service>
+
+        <service id="console.command.secrets_add" class="Symfony\Bundle\FrameworkBundle\Command\SecretsAddCommand">
+            <argument type="service" id="secret_var_processor" />
+            <tag name="console.command" command="secrets:add" />
+        </service>
+
+        <service id="console.command.secrets_edit" class="Symfony\Bundle\FrameworkBundle\Command\SecretsEditCommand">
+            <argument type="service" id="secret_var_processor" />
+            <tag name="console.command" command="secrets:edit" />
+        </service>
+
+        <service id="console.command.secrets_update_keypair" class="Symfony\Bundle\FrameworkBundle\Command\SecretsUpdateKeyPairCommand">
+            <argument type="service" id="secret_var_processor" />
+            <tag name="console.command" command="secrets:update-keypair" />
+        </service>
```

Comment:
I'm not 100% sure, but I'm thinking we could add all the services to a new secrets.xml and only load it if the config is enabled.

Reply:
okay i will investigate, that would be great